Gems/Atom/RHI/Registry/Platform/Windows/PlatformLimits.setreg

```diff
@@ -27,7 +27,7 @@
                             "DESCRIPTOR_HEAP_TYPE_DSV": [2048, 0]
                         },
                         "NumShaderVisibleCbvSrvUavStaticHandles": 2000,
-                        "AllowDescriptorHeapCompaction": true
+                        "AllowDescriptorHeapCompaction": false
```

Comment:
Discussed offline with @moudgils, his concern was the vegetation system as it the system that needed the heap compaction to work. Unfortunately the vegetation test is broken in Development, and the bindless branch shows the same behavior as mainline.

Reply:
Basically if this comes back to us saying that this PR broke large veg then we will just have to live with it until bindless support is added to RT. Tagging @mbalfour-amzn and @dmcdiarmid-ly for visibility.

Reply:
If this PR is known to break vegetation, shouldn't it be held back until bindless support is added to RT? 

There are people out there in the community that have been using the vegetation components, so at a minimum if the plan is to intentionally break it for a period of time, then at an absolute minimum this should get an "Impactful Change" notification telling them that they will just have to live with it being broken and for how long. :(

Reply:
The compaction bug occurred specifically when loading and unloading levels which consumed large amounts of descriptors, fragmenting the descriptor heap. Garbage collection was one way to fix this but moving vegetation/terrain to bindless is, IMO, the correct way to fix it (which would need this PR). However, it's possible that the GC may be needed until terrain/vegetation are ported over.

Reply:
ah, kk, sounds like there's nothing for me to see here. i'll move along. :)

Reply:
Compaction was added because of the way we created descriptors for RT buffers. It was done in chunks where we would discard the previous range and re-allocate  a new bigger range leaving a big hole behind quickly fragmenting the whole heap. So this was easy to repro if you were to create a large enough vegetation patch and in the middle of building these patches it will crash. The call stack was very obvious in indicating it is due to allocating a big range for DX12 descriptors. This was actually happening as we populate vegetation patches within the level in Edit mode. Since this was added a lot of optimizations has been added and it is possible that this crash would occur on much bigger patches. So it will be there but will be harder to repro. 

This change is **big** one and it needs to go in first before we can move RT to bindless. The other option is to fix descriptor heap compaction with the new descriptor heap scheme. This would be very technical and complete throw away work. My recommendation is that we will need to live with some amount of instability (for vegetation) until we move RT to bindless.